datalake/src/main/java/com/sequenceiq/datalake/flow/SdxReactorFlowManager.java

```diff
@@ -116,6 +119,35 @@ public FlowIdentifier triggerDatalakeRuntimeUpgradeFlow(SdxCluster cluster, Stri
         }
     }
 
+    private boolean shouldSdxBackupBePerformed(SdxCluster cluster) {
+        boolean retVal = true;
+        String reason = null;
+        if (cluster.isRangerRazEnabled()) {
+            retVal = false;
+            reason = "RAZ is enabled";
```

Comment:
**suggestion:** consider making the reason `"backup is not supported with RAZ enabled"`, I think that makes it clearer that we're explicitly _not_ supporting backups with RAZ.